packages/contracts-bedrock/test/setup/Setup.sol

```diff
@@ -78,9 +83,6 @@ contract Setup {
     /// @notice Allows users of Setup to override what L2 genesis is being created.
     Fork l2Fork = LATEST_FORK;
 
-    /// @notice Indicates whether a test is running against a forked production network.
-    bool private _isForkTest;
-
```

Comment:
don't need this var anymore, we just use the `isForkTest()` func to read the env var.